src/hCassandra_test.py

```diff
@@ -0,0 +1,181 @@
+__author__ = 'annyz'
+
```

Comment:
Here you will need to add a path to allow hydra to load properly
+from sys import path
 +path.append("hydra/src/main/python")